index.bs

```diff
@@ -673,38 +675,35 @@ authorizing an authenticator with which to complete the operation.
 <pre class="idl">
     enum Attachment {
         "platform",
-        "cross_platform"
+        "cross-platform"
     };
 </pre>
 
 <div dfn-for="Attachment">
-    Clients may communicate with authenticators using a variety of mechanisms.
-    For example, a client may use a platform-specific API to communicate
-    with an authenticator which is physically bound to a platform.  On the
-    other hand, a client may use a variety of cross-platform transport
-    protocols such as Bluetooth (see [[#transport]] to discover and communicate
-    with authenticators.  We define authenticators that are part of the
-    client's platform as having a |platform| attachment, while those that are
-    reachable via standardized cross-platform transport protocols are defined
-    as having |cross_platform| attachment.  This distinction is important
-    because there are use-cases where only |platform| attached authenticators
-    are acceptable to a [RP], and conversely ones where only |cross_platform|
-    attached authenticators are employed.  As a concrete example of the former,
-    a credential on a platform authenticator may be used by [RPS] to quickly
-    and conveniently reauthenticate the user with a minimum of friction.  As
-    a concrete example of the latter, when the user is accessing the [RP] from
-    a given client for the first time, they may be required to use a
-    cross-platform authenticator which was originally set up on a different
-    client.
+    Clients may communicate with authenticators using a variety of mechanisms. For example, a client may use a platform-specific
+    API to communicate with an authenticator which is physically bound to a platform. On the other hand, a client may use a
+    variety of standardized cross-platform transport protocols such as Bluetooth (see [[#transport]]) to discover and
+    communicate with <a>cross-platform attached</a> authenticators. We define authenticators that are part of the client's
```

Comment:
I don't know what a "cross-platform attached" authenticator is.  I don't understand why "cross-platform" was not clear here.